data/human/culture conversations.txt

```diff
@@ -103,6 +103,35 @@ mission "Trash Fire Crops"
 
 
 
+mission "Worker's Occasion"
+	minor
+	source
+		government "Republic"
+		attributes "dirt belt"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn, and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you but you recall how unreachable they would've been for you before your career change.`
+			choice
+				`	(Offer to buy the man's dinner.)`
+					goto aid
+				`	(Pity the man.)`
+					goto pity
+				`	(Smile at the man.)`
+			`	You smile at the man and reflect on the value of small achievements. He catches your smile out of the corner of his eye and gives you a hesitant but friendly wave back before you go back about your business.`
```

Comment:
I feel like should be "go back to your business" or "go about your business", not a combination of both.

Reply:
```suggestion
			`	You smile at the man and reflect on the value of small achievements. He catches your smile out of the corner of his eye and gives you a hesitant but friendly wave back before you go about your business.`
```